canonical-web-and-design/greenhouse-exporter
# Add download as csv endpoint

## Done
Add an api endpoint to download jobs data as CSV . When we refactor the backend we can use the same logic to make it applicable to different queries. 
## QA
- VPN
- `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- Go to http://localhost:8501/api/download/jobs
- Check it downloads a `csv` file with the same information that the jobs table
 ## Issue
[#100](https://github.com/canonical-web-and-design/workplace-engineering-squad/issues/100)
